packages/react-components/src/organisms/ResearchOutputFormSharingCard.tsx

```diff
@@ -135,7 +138,11 @@ const ResearchOutputFormSharingCard: React.FC<
       <LabeledDropdown<ResearchOutputType | ''>
         title="Type"
         subtitle="(required)"
-        description={`Select the option that applies to this ${documentType.toLowerCase()}.`}
+        description={`${
```

Comment:
Pass in the description directly from the parent component based on the publishing entity.